simple/utils/astrometry.py

```diff
@@ -181,6 +182,103 @@ def ingest_parallaxes(db, sources, plxs, plx_errs, plx_refs, comments=None):
     return
 
 
+def ingest_parallax(
+    db,
+    source: str = None,
+    parallax: float = None,
+    plx_error: float = None,
+    reference: str = None,
+    comment: str = None,
```

Comment:
add `raise_error` flag so user can decide if they want the script to fail or to keep going.
```python
if raise_error:
    raise
else:
    logger.warning
```